doc/OnlineDocs/contributed_packages/index.rst

```diff
@@ -23,8 +23,9 @@ Contributed packages distributed with Pyomo:
    mindtpy.rst
    satsolver.rst
    sensitivity_toolbox.rst
+   community.rst
 
 Contributed packages distributed independently of Pyomo, but accessible
 through ``pyomo.contrib``:
 
-* `pyomo.contrib.simplemodel <http://pyomocontrib-simplemodel.readthedocs.io/en/latest/source.html>`_
+* `pyomo.contrib.simplemodel <http://pyomocontrib-simplemodel.readthedocs.io/en/latest/source.html>`_
```

Comment:
I can't see the difference here, but it should reflect what is happening in master.